### ISSUE TITLE
Issue with "invalid-assertions"

### ISSUE THREAD

**Issue by ThomasAaron** [onechiporenko/eslint-plugin-mocha-cleanup#25]
I have found an issue with `invalid-assertions`.  Arrow functions without parenthesis or curly braces around the body are not detected as invalid assertions.

**Does not work:**
```
Object.entries(myReference).forEach(([key, value]) =>
        it(`prints ${key}`, () => expect(pageData.includes(value)) ),
      );
```

**Works:**
```
Object.entries(myReference).forEach(([key, value]) =>
        it(`prints ${key}`, () => { expect(pageData.includes(value)) }),
      );
```

**Works:**
```
Object.entries(getStatementConfig()).forEach(([key, value]) =>
        it(`prints ${key}`, () => ( expect(pageData.includes(value)) )),
      );
```

versions:
```
"node": "v14.16.0"
"vscode": "1.60.2"
"eslint": "^7.20.0"
"eslint-plugin-mocha-cleanup": "^1.9.1"
```

**Comment by onechiporenko** [comment]
Hi, @ThomasAaron. Thanks for the report!

**Comment by onechiporenko** [comment]
v1.10.0 is released with fix

**Comment by ThomasAaron** [comment]
@onechiporenko I can confirm after updating that the issue appears resolved. I appreciate the quick turnaround!  I hope you have a great week.
